content/lang/fr/index.md

```diff
@@ -0,0 +1,233 @@
+        Le design de Preact vous permet de réutiliser sans aucune modification des milliers de composants de l'écosystème React.
+    </p>
+    
+    <p>
+        Ajouter un simple alias <a href="/guide/switching-to-preact#how-to-alias-preact-compat">preact-compat</a> à votre bundle fournit une couche de compatibilité qui vous permettra d'utiliser les composants React les plus complexes dans votre application.
+    </p>
+</section>
+
+
+<section class="home-top">
+    <h1>Exemple</h1>
+</section>
+
+
+<section class="home-split">
+    <div>
+        <h2>Composant Liste de Tâches</h2>
+        <pre><code class="lang-js">
+export default class TodoList extends Component {
+    state = { todos: [], text: '' };
+    setText = e =&gt; {
+        this.setState({ text: e.target.value });
+    };
+    addTodo = () =&gt; {
+        let { todos, text } = this.state;
+        todos = todos.concat({ text });
+        this.setState({ todos, text: '' });
+    };
+    render({ }, { todos, text }) {
+        return (
+            &lt;form onSubmit={this.addTodo} action="javascript:"&gt;
+                &lt;input value={text} onInput={this.setText} /&gt;
+                &lt;button type="submit"&gt;Ajouter&lt;/button&gt;
+                &lt;ul&gt;
+                    { todos.map( todo =&gt; (
+                        &lt;li&gt;{todo.text}&lt;/li&gt;
+                    )) }
+                &lt;/ul&gt;
+            &lt;/form&gt;
+        );
+    }
+}
+        </code></pre>
+    </div>
+    
+    <div>
+        <h2>Démonstration</h2>
+        <pre repl="false"><code class="lang-js">
+import TodoList from './todo-list';
+
+render(&lt;TodoList /&gt;, document.body);
+        </code></pre>
+        <div class="home-demo">
+            <todo-list></todo-list>
+        </div>
+    </div>
+</section>
+
+
+<section class="home-split">
+    <div>
+        <h2>Télécharger les Etoiles Github</h2>
+        <pre><code class="lang-js">
+export default class Stars extends Component {
+    async componentDidMount() {
+        let stars = await githubStars(this.props.repo);
+        this.setState({ stars });
+    }
+    render({ repo }, { stars=0 }) {
+        let url = \`//github.com/${repo}\`;
+        return (
+            &lt;a href={url} class="stars"&gt;
+                ⭐️ {stars} Etoiles
+            &lt;/a&gt;
+        );
+    }
+}
+        </code></pre>
+    </div>
+    
+    <div>
+        <h2>Démonstration</h2>
+        
+        <pre repl="false"><code class="lang-js">
+import Stars from './stars';
+
+render(
+    &lt;Stars repo="developit/preact" /&gt;,
+    document.body
+);
+        </code></pre>
+       
+        <div class="home-demo">
+            <github-stars simple user="developit" repo="preact"></github-stars>
+        </div>
+    </div>
+</section>
+
+
+<section class="home-top">
+    <h1>Prêts pour le plongeon ?</h1>
```

Comment:
Do we say that in French? `Prêts pour le grand saut?` is another option, but dunno.
I was thinking of removing the S from `prêt` but it might be correct as we’re using `vous` the whole time!